composer/spec/fixtures/projects/v1/vcs_source_unreachable/composer.json

```
@@ -0,0 +1,12 @@
+{
+  "require": {
+      "jazz/phpunit-test-extension": "1.2.2",
+      "symfony/polyfill-mbstring": "1.0.1"
+  },
+  "repositories": [
+    {
+        "type": "vcs",
+        "url": "git@github.com:dependabot-fixtures/phpunit-test-extension.git"
```

This repo does not exist right? Can we make that more clear somehow?

```suggestion
        "url": "git@github.com:dependabot-fixtures/this-repo-does-not-exist.git"
```

---

Ah yes 👍